### PR TITLE
Fix the failure of TestTranscoder_DetectionFreq in CircleCI

### PR DESCRIPTION
**Pull request by JamesWanglf**
This is related with #301 
- Before we run `TestTranscoder_DetectionFreq` function, we need to download `tasmodel.pb` first.
- We need to set the library path for the Tensorflow linking in env variables.
- To build the ffmpeg with libtensorflow, `experimental` should be contained in BUILD_TAGS.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -45,7 +45,13 @@ jobs:
             - "/home/circleci/ffmpeg"
             - "/home/circleci/compiled"
           key: ffmpeg-cache-{{ checksum "install_ffmpeg.sh" }}
-
+          
+      - run:
+          name: "Download tasmodel.pb"
+          command: |
+            wget https://github.com/livepeer/livepeer-ml/releases/download/v0.3/tasmodel.pb
```
Comment (cyberj0g) [inline .circleci/config.yml:52]: Given `lpms` is not released separately, maybe not pin the version here and link to [latest](https://github.com/livepeer/livepeer-ml/releases/latest/download/tasmodel.pb) release, so we don't have to update CI job for every new version of the model?

File: .circleci/config.yml [hunk 2]
```diff
@@ -45,7 +45,13 @@ jobs:
...
+            mv tasmodel.pb ./ffmpeg
```
Comment (cyberj0g) [inline .circleci/config.yml:53]: Maybe download right to destination dir e.g. `wget URL -P ffmpeg`

**Review by cyberj0g — CHANGES_REQUESTED** [review]

**Comment by JamesWanglf** [comment]
If I disable `restore_cache` step, all tests will be passed.
If I enable this step, test will be failed at Test_DetectionFreq method. I guess this is occurred because the cached ffmpeg will be used, rather than the latest checkout.

**Review by oscar-davids — APPROVED** [review]
LGTM!

**Comment by cyberj0g** [comment]
LGTM!

**Review by cyberj0g — APPROVED** [review]

**Pull request merged**